### PR TITLE
Task 2: Webpack

### PR DESCRIPTION
**Pull request by DariaGunina**

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+function buildConfig() {
+    const env = process.env.NODE_ENV;
+    if (env === 'dev' || env === 'prod') {
```
Comment (mmarkelov) [inline webpack.config.js]: Not an error, but you can use try catch also:
```js
try {
  return require(`./webpack.${env}.js`);
} catch(e) {
  console.log("Wrong webpack build parameter. Possible choices: 'dev' or 'prod'.");
}
```

Comment (mmarkelov) [inline webpack.config.js]: Also this kind of strings is better to use as constant like this.
```js
const DEV = 'dev';
```

File: webpack.prod.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+                test: /\.css$/,
+                use: [MiniCssExtractPlugin.loader, 'css-loader'],
+            },
+        ],
+    },
+    optimization: {
+        minimize: true,
+        minimizer: [
+            new CssMinimizerPlugin({
+                test: /\.foo\.css$/i,
```
Comment (mmarkelov) [inline webpack.prod.js]: I think you should remove this, cause with `regexp` **css minimizer** will be used only for specific style files like `foo.css`

File: webpack.config.js (hunk 1 above)
Comment (DariaGunina) [reply webpack.config.js]: This function has been updated

Comment (DariaGunina) [reply webpack.config.js]: This is done

File: webpack.prod.js (hunk 2 above)
Comment (DariaGunina) [reply webpack.prod.js]: This is done

**Pull request merged**